utils/log.go

```diff
@@ -68,3 +72,15 @@ func Errorf(format string, args ...interface{}) {
 func Debug() bool {
 	return logLevel == LogLevelDebug
 }
+
+func init() {
```

Comment:
I can factor the implementation into a separate function and test that individually, but I don't see a clean way to test init things.